File: objcClient/objcClient/ViewController.m
//
//  ViewController.m
//  objcClient
//
//  Created by Darius Jankauskas on 01/07/2020.
//

#import "ViewController.h"
#import <objcFramework/objcFramework.h>
#import <swiftFramework/swiftFramework.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    FramyO *o = [[FramyO alloc] initWithName:@"oOOo"];
    NSLog(@"%@", o.name);
}


@end

File: swiftClient/swiftClient/ViewController.swift
//
//  ViewController.swift
//  swiftClient
//
//  Created by Darius Jankauskas on 30/06/2020.
//

import UIKit
import swiftFramework
import objcFramework

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let framy1 = FramyS(name: "kuku")
        print("\(framy1.name)")
        
        let framy2 = FramyO(name: "bubu")
        print("\(framy2.name)")
    }


}

